### PR TITLE
Change our routings to be more in line with Core

### PR DESCRIPTION
**Pull request by Jbekker**
Tests are still passing, also manually tested.

### CODE REVIEW

File: modules/product/commerce_product.links.action.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
-commerce_product.add:
-  route_name: commerce_product.product_add_page
+entity.commerce_product.product.add:
+  route_name: entity.commerce_product.product.add_page
   title: 'Add a new product'
   appears_on:
-    - commerce_product.list
+    - entity.commerce_product.product.list
```
Comment (bojanz) [inline modules/product/commerce_product.links.action.yml]: This should be entity.commerce_product.list

File: modules/price/commerce_price.routing.yml [hunk 2]
```diff
@@ -30,7 +30,7 @@ entity.commerce_currency.delete_form:
   requirements:
     _permission: 'administer commerce_store entities'
 
-commerce_price.commerce_currency.devel:
+entity.commerce_price.commerce_currency.devel:
```
Comment (bojanz) [inline modules/price/commerce_price.routing.yml]: The .commerce.price part needs to disappear.
entity.commerce_currency.devel

File: commerce.links.action.yml [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+  route_name: entity.commerce.store.add_page
+  title: 'Add a new store'
   appears_on:
-    - commerce.store_type_list
+    - entity.commerce.store.list
 
-commerce.store_add:
-  route_name: commerce.store_add_page
-  title: 'Add a new store'
+entity.commerce.store_type.add:
```
Comment (bojanz) [inline commerce.links.action.yml]: Since the pattern is entity.$entity_type.$route, this one should be entity.commerce_store_type.add

**Comment by bojanz** [comment]
We need to make sure we are following the entity.$entity_type.$route everywhere. Right now that isn't the case.

File: modules/product/src/Entity/CommerceProductType.php [hunk 4]
```diff
@@ -34,8 +34,8 @@
  *     "uuid" = "uuid"
  *   },
  *   links = {
- *     "edit-form" = "commerce_product.product_type_edit",
- *     "delete-form" = "commerce_product.product_type_delete"
+ *     "edit-form" = "entity.commerce_product.product_type.edit",
```
Comment (bojanz) [inline modules/product/src/Entity/CommerceProductType.php]: entity.commerce_product_type.edit_form

File: modules/price/commerce_price.module [hunk 5]
```diff
@@ -16,7 +16,7 @@ function commerce_price_entity_operation(EntityInterface $entity) {
   $entity_type_id = $entity->getEntityTypeId();
   if (\Drupal::currentUser()->hasPermission('access devel information') && $entity_type_id == 'commerce_currency') {
     $operations['devel'] = array(
-      'route_name' => 'commerce_price.commerce_currency.devel',
+      'route_name' => 'commerce_currency.devel',
```
Comment (bojanz) [inline modules/price/commerce_price.module]: should be 'route_name' => 'entity.commerce_currency.devel',

**Comment by damz** [comment]
That was a big one. Good job guys!

**Pull request merged**